tests/functional/spec/difference.spec.js

```diff
@@ -0,0 +1,73 @@
+import AgeBlockYearPage from "../generated_pages/difference_in_years/age-block.page";
+import AgeBlockMonthYearPage from "../generated_pages/difference_in_years_month_year/age-block.page";
+import AgeBlockDayMonthYearRangePage from "../generated_pages/difference_in_years_range/date-block.page";
+import AgeBlockMonthYearRangePage from "../generated_pages/difference_in_years_month_year_range/date-block.page";
+
+describe("Difference check (years)", () => {
+  before("Load the survey", () => {
+    browser.openQuestionnaire("test_difference_in_years.json");
```

Comment:
I think while we are here, we should rename the schema to group them by feature like we do already. So i.e `test_placeholder_difference_in_years.json` or using actual name of the transform `test_placeholder_calculate_date_difference.json`

Reply:
Changed them to option 1, so `test_placeholder_difference_in_years.json` etc.